docs/README.md

```diff
@@ -0,0 +1,352 @@
+<!-- markdownlint-disable-next-line first-line-h1 -->
+![https://detdigitalefolkebibliotek.dk/sites/default/files/ddf_logo_rgb_blue_web.png](https://detdigitalefolkebibliotek.dk/sites/default/files/ddf_logo_rgb_blue_web.png)
```

Comment:
Please move the logo to the root README. It is nice to have here to show who owns the project but in the context of the documentation site it is irrelevant.